docs/serverless-CR.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Serverless
+---
+
+The `operator.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the Serverless configuration that you want to install on your cluster. To get the up-to-date CRD and show the output in the YAML format, run this command:
+
+   ```bash
+   kubectl get crd operator.kyma-project.io -o yaml
+   ```
+
+## Sample custom resource
+
+The following Serverless custom resource (CR) shows configration of Serverless with the external registry and two user URLs for Publisher Proxy and Trace Collector.
+
+   ```yaml
+   apiVersion: operator.kyma-project.io/v1alpha1
+   kind: Serverless
+   metadata:
+     finalizers:
+     - serverless-manager.kyma-project.io/deletion-hook
+     name: default
+     namespace: kyma-system
+   spec:
+     dockerRegistry:
+       enableInternal: false
+       secretName: my-secret
+     eventPublisherProxyURL: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     traceCollectorURL: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   status:
+     conditions:
+     - lastTransitionTime: "2023-04-28T10:09:37Z"
+       message: Configured with default Publisher Proxy URL and default Trace Collector
+         URL.
+       reason: Configured
+       status: "True"
+       type: Configured
+     - lastTransitionTime: "2023-04-28T10:15:15Z"
+       message: Serverless installed
+       reason: Installed
+       status: "True"
+       type: Installed
+     eventPublisherProxyURL: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     state: Ready
+     traceCollectorURL: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   ```
+
+## Custom resource parameters
+
+For details, see the [Serverless configuration file](https://github.com/kyma-project/serverless-manager/blob/main/api/v1alpha1/serverless_types.go).
+
+| Parameter         | Description                                   |
+| ---------------------------------------- | ---------|
+| **spec.eventPublisherProxyURL** | URL for your Event Publisher Proxy |
```

Comment:
Can we generalize this description a little bit? the `eventPublisherProxyURL` is not correct and we will change it in the next PRs. This field doesn't have anything in common with the proxy. [Here](https://github.com/kyma-project/serverless-manager/issues/100) is ticket to change these fields (eventPublisherProxyURL and traceCollectorURL).
```suggestion
| **spec.eventPublisherProxyURL** | Event publisher endpoint used by [the Serverless SDK](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-08-function-specification/#event-object-sdk) |
```